addons/website_sale/models/product_template.py

```diff
@@ -274,13 +274,6 @@ def _get_combination_info(self, combination=False, product_id=False, add_qty=1,
                 combination_info['price_extra'], product_taxes, taxes, company_id, pricelist,
                 product, partner
             )
-            base_unit_price = product._get_base_unit_price(list_price)
```

Comment:
`_get_base_unit_price` could be used below when updating `combination_info`.
Also, this could be a CLN instead of a REM IMHO (we do not drop a feature/module/...)

Reply:
True. Shouldn't we use the base_unit_price below in the update then, to take into account the currency conversion?

Reply:
I don't think it's necessary, AFAIK the `list_price` in `combination_info` provided by the super call is already given in the pricelist currency.